data/content_scripts/posting_process/context_menu.js

```diff
@@ -35,7 +35,7 @@ postingProcess.menuClickHandler = function(node, data) {
     text: selectionText,
     pageURL: window.location.href,
   };
-  self.postMessage(info);
+  self.postMessage({name: "startPostingProcess", content: info});
```

Comment:
Regarding the second point, the listener function runs in the background context?

(side note, [stray comma](https://github.com/privly/privly-jetpack/blob/master/lib/posting_process.js#L377))